notebooks/234-blip/Untitled.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Should we mention what all those parameters to export function mean? Like why we are specifying them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>yes, I'm working on that right now</p>

Reply:
added